evalml/automl/automl_search.py

```diff
@@ -185,13 +188,25 @@ def __init__(self,
         else:
             raise TypeError("max_time must be a float, int, or string. Received a {}.".format(type(max_time)))
 
+        if _max_batches:
+            if not max_batches:
+                max_batches = _max_batches
+            logger.warning("`_max_batches` will be deprecated in the next release. Use `max_batches` instead.")
```

Comment:
RE comments above, let's delete this, and `_max_batches`, entirely, in favor of `max_batches`